src/api/marking/marking.router.ts

```diff
@@ -45,7 +43,7 @@ markingRouter.post(
 markingRouter.route("/critter/:id").get(
   catchErrors(async (req: Request, res: Response) => {
     // validate marking id and confirm that marking exists
-    const id = isUUID(req.params.id);
+    const { id } = uuidParamsSchema.parse(req.params);
     const markings = await getMarkingsByCritterId(id);
     if (!markings.length) {
```

Comment:
I think we can remove this catch, and just return [] for no markins for a critter. getMarkingsByCritterId should throw an error if critter_id is not found.